frontend/src/views/Contributors.vue

```diff
@@ -0,0 +1,56 @@
+<template lang="pug">
+  div
+    h1 Contributors
+    h2(v-if="this.errorMessage") {{ errorMessage }}
```

Comment:
For SEO purposes, `h` tags should never be used for anything other than semantic titles and headers.
Can you please use a simple `div` or `p` tag with a classname here, instead?

Reply:
Sure.